jooq/src/main/java/com/cerner/beadledom/jooq/ThreadLocalCommitHookExecutingTransactionListener.java

```diff
@@ -25,7 +25,7 @@
   }
```

Comment:
Do we need to update changelog?

Reply:
Yup, added https://github.com/cerner/beadledom/pull/158/commits/427bc17abf71852fffbfd5c6f56da70684509133